src/Symfony/Component/Security/Http/RequestDecision.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http;
+
+class RequestDecision
+{
+    public bool $support;
+    public ?bool $lazy = null;
```

Comment:
when is this used? what could be built with that?
```suggestion
    public bool $isLazy;
```

Reply:
I planned to use this class for firewall listeners.

Reply:
Oh, so let's remove it from this PR and add it in the one about listeners then.

Reply:
Should the `TraceableAuthenticator` set them then?